packages/langium/src/grammar/langium-grammar-validator.ts

```diff
@@ -308,6 +309,18 @@ export class LangiumGrammarValidator {
         }
     }
 
+    checkFragmentsInTypes(grammar: ast.Grammar, accept: ValidationAcceptor): void {
+        grammar.types.map(e => {
```

Comment:
Both `map`s here can be replaced with `forEach` since we don't need a "mapped value".